.github/workflows/assign_failed_test_project.yml

```diff
@@ -0,0 +1,21 @@
+name: Auto Assign Failed Test Issue to Solid Tests Project
+
+on:
+  issues:
+    types: [created]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_bench_test_project:
+    runs-on: ubuntu-latest
+    name: Assign to Solid Tests Project
+    steps:
+    - name: Run issues assignment to project Solid Tests Kanban
+      uses: srggrs/assign-one-project-github-action@1.2.0
+      if: |
+        github.event.issue.user.login == 'sre-bot' &&
```

Comment:
Does it mean only issues created by bot will be added into the target project?

Reply:
yes. Issues create by sre-bot with `bench-test` label are done by our daily report job, so these issues can be auto-added to the `Solid Tests`. I'm not sure if there are some other conditions, we should also do this. @mahjonp PTAL

Reply:
Why can't bot do it directly?